docs/services.md

```diff
@@ -11,8 +11,20 @@ aliases:
 
 Services are built-in software packages that make it easier to add complex capabilities such as motion planning or object detection to your machine.
 
+{{< imgproc src="/viam/machine-components.png" alt="Machine components" resize="1000x" style="max-width:650px" >}}
+<br>
+
+Even though many services run locally within `viam-server`, you can think of them as discrete building blocks; you can run your machine using none, some, or all of them, depending on your use case.
```

Comment:
"Even though" is generally used to introduce a seeming contradiction - I don't think there's a contradiction here.

```suggestion
You can think of services as discrete building blocks; you can run your machine using none, some, or all of them, depending on your use case.
```

Reply:
Yes--I do see it as a contradiction because it's software that is already there, installed on your robot, as part of RDK; you're not actually "adding" anything--you're technically just choosing to use more of what is already there on your robot.

Reply:
Ok - if I stumble over that I am not sure it's clear enough. Can you try to make it clearer? Here's my attempt but feel free to change:
```suggestion
Many builtin services run locally within `viam-server`, but you can think of them as discrete building blocks that you can mix and match however you want; you can run your machine using none, some, or all of them, depending on your use case.
```
I think this still keeps the contradiction but for people who don't see it as a contradiction (I can see something as building blocks even if its within something else...) it might be better? Open to alternatives

Reply:
```suggestion
Many built-in services run locally within `viam-server`, but you can think of them as discrete building blocks that you can mix and match however you want; you can run your machine using none, some, or all of them, depending on your use case.
```